actors/miner/src/lib.rs

```diff
@@ -2230,6 +2220,48 @@ impl Actor {
 
         Ok(ProveCommitSectorsNIReturn { activation_results: validation_batch })
     }
+    /// Returns the maximum termination fee calculation for a given initial pledge and power amount
+    fn max_termination_fee(
+        rt: &impl Runtime,
+        params: MaxTerminationFeeParams,
+    ) -> Result<MaxTerminationFeeReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let reward_smoothed = request_current_epoch_block_reward(rt)?.this_epoch_reward_smoothed;
+        let quality_adj_power_smoothed =
+            request_current_total_power(rt)?.quality_adj_power_smoothed;
+        let fault_fee = pledge_penalty_for_continued_fault(
+            &reward_smoothed,
+            &quality_adj_power_smoothed,
+            &params.power,
+        );
+
+        let max_fee = pledge_penalty_for_termination(
+            &params.initial_pledge,
+            TERMINATION_LIFETIME_CAP * EPOCHS_IN_DAY,
+            &fault_fee,
+        );
+
+        Ok(MaxTerminationFeeReturn { max_fee })
+    }
+
+    /// Returns the miner's total initial pledge amount
+    fn initial_pledge(rt: &impl Runtime) -> Result<InitialPledgeReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+
+        Ok(InitialPledgeReturn { initial_pledge: state.initial_pledge.clone() })
+    }
+
+    /// Returns the network's termination fee percentage
+    fn termination_fee_percentage(
```

Comment:
TODO: remove per the discussion on the FIP (so we don't forget).

Reply:
Removed in https://github.com/filecoin-project/builtin-actors/pull/1639/commits/a3ba05cf83918e7f247bd9a440afc3b2d7a4d20b; I'll revert if @schwartz10 considers this export a must-have.